langchain/google_search.py

```diff
@@ -0,0 +1,135 @@
+"""Chain that calls Google Search."""
+import os
+import sys
+from typing import Any, Dict, List, Optional
+
+from googleapiclient.discovery import build
```

Comment:
this should probably go inside the function (so its not a requirement for the package)